.github/workflows/cleanup.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: "Clean up deployment"
+on:
+  pull_request:
+    # run on pull requests targeting master
+    branches:
+      - master
+    types: closed
```

Comment:
didn't know this was an option!!🏆

Reply:
So this deleted the namespace when the PR was closed (i closed it multiple times to test on this PR)

Reply:
ah, thanks for clearing that up!